src/annotator/config/settings.js

```diff
@@ -56,7 +61,7 @@ export function settingsFrom(window_) {
       return null;
     }
 
-    return jsonConfigs.annotations || annotationsFromURL();
+    return checkIfString(jsonConfigs.annotations) || annotationsFromURL();
```

Comment:
Adding this check avoids downstream code from needing to deal with the case where `annotations` is not a string or `null`. Same for other functions in this file.